flix-core/src/main/java/com/epam/mentoring/flixcore/service/impl/MovieServiceImpl.java

```diff
@@ -12,9 +12,13 @@
 @Service
 public class MovieServiceImpl implements MovieService {
 
-    @Autowired
     private MovieRepository movieRepository;
 
+    @Autowired
```

Comment:
If we would use lombok it would be even easier to inject dependencies

Reply:
but up to you, it wont hurt to add lombok as it is great tool, if you will keep in mind that you will need to install IJ plugin or it will show a lot of errors.

Reply:
Added AccessDeniedHandler, AuthenticationSuccessHandler, and Lombok